README.md

```diff
@@ -102,30 +102,32 @@ Log into your `OFQA` sandbox and edit the label of the `Car` object. Change it t
 
 Once that workflow has completed, create another pull request to merge the changes from the branch `sandbox/qa` into `sandbox/uat`. You'll notice that the changes can be merged, so press the **Merge** button, and wait for the automatic deployment workflow to trigger again. Once that workflow run is complete, you'll notice that the `Car` object in the sandbox `OFUAT` has been updated with the changes from the other sandbox, and that the changes already in `OFUAT` were retained too.
 
-If you'd like to merge the changes back into the `OFQA` sandbox, then you simply need to create and merge another pull request (this time from the branch `sandbox/uat` into `sandbox/qa`), and then run the **Flow out an environment** workflow.
+If you'd like to merge the changes back into the `OFQA` sandbox, then you simply need to create and merge another pull request (this time from the branch `sandbox/uat` into `sandbox/qa`), and then run the **Deploy metadata** workflow.
 
 ## Enumerating environments
 
-The **Flow in all environments** workflow shows an example of enumerating environments in your stack and performing some work (in this case an inbound flow) for a subset of them in parallel using a GitHub Actions matrix job in your workflow.
+The **Commit Salesforce metadata changes for all environments** workflow shows an example of enumerating environments in your stack and performing some work (in this case an inbound flow) for a subset of them in parallel using a GitHub Actions matrix job in your workflow.
 
-You may have other jobs that you want to run against multiple environments only known at runtime. You can take advantage of the `env:list` command to retrieve all of the environment names in a stack and use a matrix strategy to run a job once for each environment.
+You may have other jobs that you want to run against multiple environments only known at runtime. You can take advantage of the `env:list --output=json` command to retrieve all of the environment names in a stack and use a matrix strategy to run a job once for each environment. 
 
-The trick here is to use a tool called [jq](https://stedolan.github.io/jq/) to transform the output from `env:list` into the format that GitHub Actions expects for a matrix definition. The correct jq syntax for this is `[.[] | .name]`. On top of this, `jq` offers a robust filtering syntax, for example if you only want to run your job on environments matching certain criteria.
+The trick here is to use the `--nameOnly` switch and a tool called [jq](https://stedolan.github.io/jq/) to transform the output from `env:list --nameOnly --output=json` into the format that GitHub Actions expects for a matrix definition. The correct jq syntax for this is `env:list --nameOnly --output=json | jq '.' -c`. On top of this, `jq` offers a robust filtering syntax, for example if you only want to run your job on environments matching certain criteria.
```

Comment:
> The correct jq syntax for this is `env:list --nameOnly --output=json | jq '.' -c`

Wording there is a bit awkward, maybe remove the word "jq" now, or rephrase to something like "the simplest way to combine an OrgFlow command and `jq` in this way is".